README.md

```diff
@@ -71,6 +69,34 @@ or HCL module
 $ inframap generate --hcl ./my-module/ | graph-easy
 ```
 
+## Which is the difference with `terraform grah`
+
+[Terraform Graph](https://www.terraform.io/docs/commands/graph.html) outputs a dependency graph of all the resources. We try to go one step forward,
+if the Provider is one of the ones we support we apply a set of rules to make the output more useful by showing the most important information.
+
+If the Provider is one of the ones we do now support, the output is much more close to the Terraform one, but we focus on resources so the Provider/Vars
```

Comment:
On the other hand, if the provider is not supported, the output will be closer to the Terraform one (without displaying provider / variable nodes)